look-and-feel.md

```diff
@@ -0,0 +1,65 @@
+# Look and Feel
+
+### Name and wordmark
+
+Our name uses title case with the following variations:
+
+> **HYPHA**  
+> **HYPHA** Worker Co-operative  
+> Hypha Worker Co-operative Inc. [our legal name]  
+
+We use the following wordmark in online spaces, services and social media where we represent the organization (e.g., [Twitter](https://twitter.com/hyphacoop) and [LinkedIn](https://ca.linkedin.com/company/hyphacoop)). 
+
+<img src="images/hypha-wordmark.png" alt="Hypha Wordmark in white on purple background" width="50%"/>
+
+### Typography
+
+Our **heading typeface** is [Work Sans](https://fonts.google.com/specimen/Work+Sans) and our **secondary (body) typeface** is !!!!. For the web we prefer using the [system font stack](https://css-tricks.com/snippets/css/system-font-stack/) sans-serif typeface.
```

Comment:
Proxima Nova could be listed as a secondary typeface for now if we go by the deck

Reply:
Just realized this isn't a free font :flushed: